DataAccess/DataAccess.Postgre/PostgreRepository.cpp

```diff
@@ -34,7 +34,11 @@ std::vector<Network::MessageInfo> PostgreRepository::getMessageHistoryForUser(co
                                    ->join(Utility::SQLJoinType::J_INNER, "channel_msgs", "channel_msgs.msg_id = msgs.msg_id")
                                    ->Where("channel_msgs.channel_id = " + std::to_string(channelID))
                                    ->execute();
-    
+
+    pTable->changeTable("users");
+    auto logins = pTable->Select()->columns({"login"})->execute();
```

Comment:
Do we need to grab the whole table? Can't we do something like " SELECT login WHERE user-id = sender.id" ?

Reply:
that would make line 52 much cleaner